src/openms/source/TRANSFORMATIONS/FEATUREFINDER/FeatureFinderIdentificationAlgorithm.cpp

```diff
@@ -72,6 +72,9 @@ namespace OpenMS
     defaults_.setValue("debug", 0, "Debug level for feature detection.", ListUtils::create<String>("advanced"));
     defaults_.setMinInt("debug", 0);
 
+    defaults_.setValue("extract:batch_size", 1000, "Nr of peptides for which chromatograms are extracted at the same time."
```

Comment:
"Nr of peptides used in each batch of chromatogram extraction. Smaller values decrease memory usage but increase runtime."